plugins/org.eclipse.elk.alg.topdownpacking/src/org/eclipse/elk/alg/topdownpacking/WhitespaceEliminationStrategy.java

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Kiel University and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0  
+ *******************************************************************************/
+package org.eclipse.elk.alg.topdownpacking;
+
+import org.eclipse.elk.core.alg.ILayoutPhase;
+import org.eclipse.elk.core.alg.ILayoutPhaseFactory;
+
+/**
+ * @author mka
+ *
+ */
+public enum WhitespaceEliminationStrategy
+        implements ILayoutPhaseFactory<TopdownPackingPhases, GridElkNode> {
+    WHITESPACE_ELIMINATOR;
```

Comment:
This enum has only one value. Why is it necessary?

Reply:
Other white space elimination strategies are possible and may be implemented at some point.